docs/SUMMARY.md

```diff
@@ -88,19 +88,158 @@
   - [OpenTracing](opentracing.md)
   - [Database Schemas](development/database_schema.md)
   - [Experimental features](development/experimental_features.md)
-  - [Synapse Architecture]()
-    - [Log Contexts](log_contexts.md)
-    - [Replication](replication.md)
-    - [TCP Replication](tcp_replication.md)
-  - [Internal Documentation](development/internal_documentation/README.md)
+  - [Synapse Architecture](development/synapse_architechture.md)
+    - [Overview]()
+      - Flows of data through Synapse
+        - Starts with a Request/BG Job/Module
+        - Processing
+        - Maybe Response
```

Comment:
Others should note that the non-markdown links have no effect in `SUMMARY.md`, and are instead presented here as ideas for headings in pages / general notes.

They are left here to give one an idea of what a page may contain, and will be removed/transferred to doc pages before this PR merges.